File: app/src/main/java/com/botijasoftware/utils/GlobalOptions.java
package com.botijasoftware.utils;


import android.content.SharedPreferences;

public class GlobalOptions {
	
	GlobalOptions(SharedPreferences preferences) {
		mSharedPreferences = preferences;
		mSharedPreferenceEditor = preferences.edit();
		loadPreferences();
	}


	public void loadPreferences() {
	}
	
	public void savePreferences() {
	}
	
	protected SharedPreferences mSharedPreferences;
	protected SharedPreferences.Editor mSharedPreferenceEditor;

}

File: app/src/main/java/com/botijasoftware/utils/GlobalOptions.kt
package com.botijasoftware.utils


import android.content.SharedPreferences

class GlobalOptions internal constructor(protected var mSharedPreferences: SharedPreferences) {

    protected var mSharedPreferenceEditor: SharedPreferences.Editor

    init {
        mSharedPreferenceEditor = mSharedPreferences.edit()
        loadPreferences()
    }


    fun loadPreferences() {}

    fun savePreferences() {}


}
